### ISSUE TITLE
portland housing data

### ISSUE THREAD

**Issue by ungoldman** [max-mapper/messages#40]
Hi Max,

**tl;dr:** Catherine from [Hack Oregon](http://www.hackoregon.org/) and Connor from the [HUD](http://hud.gov) PDX field office want to invite you and me to lunch and talk about quality and availability of housing data in Portland.

Connor wants to do something in Portland similar to the upcoming [Seattle Housing Hackathon](https://www.eventbrite.com/e/hack-housing-empowering-smarter-decisions-a-weekend-hackathon-registration-15310832111) but more focused on helping low-income families and the homeless (paraphrasing, Connor can elaborate). Catherine is working on a "storytelling with data" project about urban development in coordination with the U of O Journalism School. They want to invite both of us to lunch some time in the next couple of weeks to talk more about it. I'm going to try to help both projects through Code for Portland. Let me know if this is something you'd be into.

Also here is a portrait of a tsarist cat.

![](http://blogs.artinfo.com/artintheair/files/2013/08/hermitage-cat-portraits-5.jpg)

**Comment by ungoldman** [comment]
@Caged ^ I know you've spent some time messing with Portland housing datasets. The folks involved in the projects above could use some guidance as to what's out there and what's useful for what they're doing. I bet you've got some pretty good insight if you've got a little time to spare.

**Comment by max-mapper** [comment]
@ngoldman im down pretty much any time, lemme know when

**Comment by caged** [comment]
@ngoldman Sounds interesting.  Most of the work I've done has been with the building dataset (e.g. data on the physical structures),  but happy to help if I can.

**Comment by ungoldman** [comment]
@maxogden @Caged cool, thanks. It looks from their end that schedules will align around:
- Thursday, 2/12
- Friday, 2/13
- Monday, 2/16
- Wednesday, 2/18
- Friday, 2/20

/cc @catnik

**Comment by caged** [comment]
The 12th might be a little difficult for me, but the other dates look fine.

**Comment by catnik** [comment]
13th and the 20th are great for me.

On Mon, Feb 2, 2015 at 2:55 PM, Justin Palmer notifications@github.com
wrote:

> The 12th might be a little difficult for me, but the other dates look
> fine.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/maxogden/messages/issues/40#issuecomment-72556693.

## 

Catherine Nikolovski
Executive Director
503 407 2963

**Comment by catnik** [comment]
I've got a phone meeting with Julia Thompson tomorrow, director of Portland
Planning and Sustainability.  If we can get a date nailed down I'll invite
her too.

On Mon, Feb 2, 2015 at 3:00 PM, Catherine Nikolovski <
cat.nikolovski@gmail.com> wrote:

> 13th and the 20th are great for me.
> 
> On Mon, Feb 2, 2015 at 2:55 PM, Justin Palmer notifications@github.com
> wrote:
> 
> > The 12th might be a little difficult for me, but the other dates look
> > fine.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/maxogden/messages/issues/40#issuecomment-72556693.
> 
> ## 
> 
> Catherine Nikolovski
> Executive Director
> 503 407 2963

## 

Catherine Nikolovski
Executive Director
503 407 2963

**Comment by ungoldman** [comment]
I vote 13th because it is the spookiest.

**Comment by connorjmcd** [comment]
The 13th works for me.  I can meet during the day or at night.  I'm open.

**Comment by max-mapper** [comment]
13th is good for me

**Comment by connorjmcd** [comment]
Sounds like we are all in for the 13th.  Headed up to Seattle now.  Of interest, here are a few data links from HUD for the Zillow Hackathon:  http://hudegis.github.io/  & http://zillowhack.hud.opendata.arcgis.com/

**Comment by ungoldman** [comment]
As @connorjmcd says, I believe we have reached a quorum and settled on the 13th as the optimum date. If it pleases the court I would like to offer the following suggestions for our meeting:
- Meet at Esri office in downtown PDX at noon on Friday the 13th, get food at nearby food carts, adjourn to a conference room and talk things through
- Something else?

**Comment by max-mapper** [comment]
bullet point 1 sounds good

**Comment by caged** [comment]
Meeting at Esri :+1:

**Comment by catnik** [comment]
Awesome. Noon?

On Mon, Feb 2, 2015 at 3:00 PM, Catherine Nikolovski <
cat.nikolovski@gmail.com
<javascript:_e(%7B%7D,'cvml','cat.nikolovski@gmail.com');>> wrote:

> 13th and the 20th are great for me.
> 
> On Mon, Feb 2, 2015 at 2:55 PM, Justin Palmer <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> 
> > The 12th might be a little difficult for me, but the other dates look
> > fine.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/maxogden/messages/issues/40#issuecomment-72556693.
> 
> ## 
> 
> Catherine Nikolovski
> Executive Director
> 503 407 2963

## 

Catherine Nikolovski
Executive Director
503 407 2963

## 

Catherine Nikolovski
Executive Director
503 407 2963

**Comment by connorjmcd** [comment]
I'll be there.  Noon.  Friday 13th @ ESRI.  
Without conflating this discussion wanted to get your thoughts on this City SDK concept:  w/http://t.co/NJvkUSb3Jp

**Comment by catnik** [comment]
For Friday's lunch, I also invited Caleb Winter (strategy planner @TriMet)
and Andrew DeVigal (UofO School of Journalism) and Erin Aigner (Content
strategist @Second Story, but also a GIS/ maps person).  Should be a great
room, looking forward to it!

Here's an event I just made live: you are all invited, and feel free to
invite more great people if the spirit moves you.

storymakers.eventbrite.com  http://storymakers.eventbrite.com

On Mon, Feb 2, 2015 at 2:22 PM, Nate Goldman notifications@github.com
wrote:

> @maxogden https://github.com/maxogden @Caged https://github.com/Caged
> cool, thanks. It looks from their end that schedules will align around:
> - Thursday, 2/12
> - Friday, 2/13
> - Monday, 2/16
> - Wednesday, 2/18
> - Friday, 2/20
> 
> /cc @catnik https://github.com/catnik
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/maxogden/messages/issues/40#issuecomment-72551181.

## 

Catherine Nikolovski
Executive Director
503 407 2963

**Comment by ungoldman** [comment]
Thanks @catnik. That's already 8 people total so I'd like to set that as the limit to make sure the conversation is productive and everyone is able to participate.

**Comment by catnik** [comment]
Got it ;)

On Tue, Feb 10, 2015 at 10:58 AM, Nate Goldman notifications@github.com
wrote:

> Thanks @catnik https://github.com/catnik. That's already 8 people total
> so I'd like to set that as the limit to make sure the conversation is
> productive and everyone is able to participate.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/maxogden/messages/issues/40#issuecomment-73759978.

## 

Catherine Nikolovski
Executive Director
503 407 2963

**Comment by connorjmcd** [comment]
Great meeting everyone on Friday.  

Here are a few links to follow up on our discussion:
- Hack Pad for "Hack Housing: Empowering Better Decisions": https://hack-housing.hackpad.com/ (click 'Event Results' to see all 30+ projects and their GitHub repository)
- Citizenville: How to Take Town Square Digital and Reinvent Government": this book illustrates numerous examples of open data projects, that were at first faced with skepticism, but the products quickly showed value, and the cynics turned into champions: http://www.amazon.com/Citizenville-Square-Digital-Reinvent-Government/dp/0143124471 

Looking forward to seeing where this takes us...(Nate once you create a repository, I'll start plugging in data sets.)

**Comment by ungoldman** [comment]
Hi everybody! Thanks again for taking the time to meet last Friday. I've made a repository on Github to help gather housing datasets and information on the different projects we talked about.

> https://github.com/CodeForPortland/housing

I've thrown together a starting point but there's plenty to add. Feel free to [edit the README](https://github.com/CodeForPortland/housing/edit/master/README.md) and open a pull request to add or change what's there, or [create an issue](https://github.com/CodeForPortland/housing/issues/new) to discuss something or ask if you want help adding or changing anything. Feel free to email me directly with questions, criticisms, demands, etc.

**Comment by catnik** [comment]
We've finally got a lot of our list of projects and roles more clearly defined for the workshop.  

Here's the google doc with all the basic info.  Feel free to share it! 

https://docs.google.com/document/d/1VL_xrg5We_3yRdehWUYN548W63gARjoQiyD5QO0aCZs/edit?usp=sharing

Justin, Max--- I wanted to follow up to see if/where/how you would like to be involved.  (There are a few team spots I didn't list that I think might be a good fit for each of you, but I'm happy anywhere you feel inspired) 

If you're interested, there's a more in-depth doc I can share with very specific project information and running team lists. 

Actually I can just tell you the media partners now: 

Earthquake: OPB
Innovation Economy: Portland Business Journal
Education: Oregonian
Campaign Finance: Willamette Week
Sustainable Housing: Portland Mercury
Story of the Story: W+K and Premier Press

**Comment by ungoldman** [comment]
Closing this as everything has happened. Thanks all!
